### DISCUSSION TITLE
What are usefull standard links, part of each docu, for users to know about?

### DISCUSSION THREAD

**Discussion by timea-solid** [solid/solidos#65] [Archive] [answered]
I'd like to collect and discuss useful links for different users when they land on readmes on GitHub or first page of wikis on GitHub or...

Proposed standard links part of all documentations:

- for learning about Solid: [about Solid](https://github.com/solid/solid) and [solidproject.org](https://solidproject.org);
- to try out Solid: [getting started with Solid](https://solidproject.org/developers/tutorials/getting-started);
- for the SolidOS user guide: [visit user guide](https://github.com/solid/userguide);
- for the SolidOS code: [visit SolidOS GitHub](https://github.com/solid/solidos);
- for the SolidOS stand-alone desktop app visit: [Data-Kitchen](https://github.com/solid/data-kitchen);
- for how-tos around how the community works: [solid process](https://github.com/solid/process);
- chat with others about Solid: [forum](https://forum.solidproject.org/) and [instant chats](https://gitter.im/solid/home);
- join an event: [Solid events](https://solidproject.org/events);
- community code of conduct: [Solid CoC](https://github.com/solid/process/blob/main/code-of-conduct.md).

Further links:
- for SolidOS:
    - [SolidOS FAQs](https://github.com/solid/solidos/wiki/FAQs)
    - [SolidOS developer guide](https://github.com/solid/solidos/wiki)
    - report a problem for SolidOS by [creating a git issues](https://github.com/solid/solidos/issues)
- for Solid in general:
    - have a new idea? Create a Solid [user story/new idea ticket](https://github.com/solid/user-stories)
    - latests [Solid specification](https://github.com/solid/solid-spec) and [overview](https://solid.github.io/specification/)
    - [Solid history](https://gitlab.com/solid.community/support/-/wikis/Home/solid.community#solidcommunity)

**Comment by ewingson** [comment]
months ago I brainstormed.
boiled down to 7 links.

* github start page
* github spec
* solidproject.org
* gitter chat
* app list
* forum
* solid idp list

**Comment by timea-solid** [comment, marked as answer]
The final list is presented on solidOS readme: https://github.com/solid/solidos
